test/collectionUtils/AnyResourceTransformer.test.ts

```diff
@@ -90,4 +92,13 @@ describe("AnyResourceTypeTransformer", () => {
       "/tmp/FHIR/AnyResource"
     );
   });
+
+  it("should keep the IResource import if the interface extends IResource", () => {
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5970468)